docs/mine/lotus/miner-wallets.md

```diff
@@ -0,0 +1,125 @@
+breadcrumb: 'Miner Wallets'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+During miner initialization, a miner actor is created on the chain. This actor gives the miner its ID (and looks like `t0...`). The miner actor be in charge of collecting all the payments sent to the miner (as identified by its ID).
+
+The Lotus Miner daemon, in turn, performs the operations as required by the network during and can use different Lotus Node wallets to pay the fees or interact with the Miner actor. Information on how to manage Lotus wallets [can be found here](../../get-started/lotus/send-and-receive-fil.md).
+
+The different types of wallets associated to a miner are described below:
+
+[[TOC]]
+
+## The owner address
+
+The owner address corresponds to a Lotus Node wallet address that is provided during the [miner initialization](miner-setup.md). It is a privilegdged address since it is the only one able to withdraw funds from the miner actor and pay for any miner-actor-triggered changes.
+
+The owner address will be used as worker and control address if none are defined.
+
+Given the high value of this wallet, we recommend using a separate worker address and keeping the owner address offline as a cold wallet.
+
+## The worker address
+
+Then provided, the worker address is used to sign blocks and submit proofs associated with this miner (otherwise the owner address is used). It must be a BLS address and can be specified during `init` with the `--worker` flag.
+
+The worker address is used to sign and send various types of messages (WHICH ONES? including the createStorageMiner message), which means it needs to pay for the gas of those messages. Make sure you keep an eye on its balance!
+
+## Control addresses
+
+Control addresses can be used to pay for WindowPoSt messages (ONLY?).
+
+WindowPoSt is the mechanism through which storage is verified in Filecoin and which requires miners to submit proofs for all sectors every 24h. Those proofs are submitted as messages to the chain which pay the respective fees.
+
+Because many other mining related actions require sending messages to the chain, and not all of those are "high value", it may be desirable to use a separate account to send PoSt messages from. This allows for setting lower GasFeeCaps on the lower value messages without creating head-of-line blocking problems for the PoSt messages in congested chain conditions.
+
+To set this up, first create a new wallet, and send it some funds to it for gas fees:
+
+```sh
+lotus wallet new bls
+t3defg...
+
+lotus send t3defg... 100
+```
+
+Next add the control address:
+
+```sh
+lotus-miner actor control set --really-do-it t3defg...
+Add t3defg...
+Message CID: bafy2..
+```
+
+Wait for the message to land on chain:
+
+```sh
```

Comment:
FYI, this is correct as written, but when we switch to actors v1, I think there will be one more step -- a `ConfirmUpdateWorkerKey` message that needs to be sent. Lotus doesn't currently have a CLI for it (since we don't actually use v1 yet).

Mostly just flagging to make a note of it somewhere.